pkg/apis/core/types.go

```diff
@@ -4108,8 +4110,46 @@ type ResourceQuotaSpec struct {
 	// If not specified, the quota matches all objects.
 	// +optional
 	Scopes []ResourceQuotaScope
+	// Priority at which this quota object is scoped
+	// +optional
+	Priority *ScopeSelector
```

Comment:
should this be `ScopeSelector` rather than `Priority`

Reply:
ok, i think this should be `scopeSelector` since it has backwards compatibility with prior scopes via exists operator.